oss-cloud-frontend/src/components/AddContributor.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-dialog v-model="dialog" max-width="600px">
```

Comment:
Avoid inline CSS

Reply:
For some reason, it doesn't work with external CSS.